controllers/auth.controller.js

```diff
@@ -0,0 +1,53 @@
+const authService = require("../services/auth.service");
+require("dotenv").config();
+
+const registerUser = async (req, res) => {
+
+  try {
+    if (JSON.stringify(req.body)==="{}") {
```

Comment:
JSON.stringify is a costly function. you can use Object.keys which is much faster.

Reply:
Yes, I have used Object.keys.